feature/featuretags/featuretags.go

```diff
@@ -32,37 +34,89 @@ func (ft FeatureTag) OmitTag() string {
 	return "ts_omit_" + string(ft)
 }
 
+// DepSet returns the set of features that must be included to
```

Comment:
(non-blocking)  Do we foresee needing dependencies in another format? Could this be plain `Deps` (or maybe better `Requirements`? People never agree on the distinction between "dependencies" and "dependents" and we keep having to stipulate which direction we mean).

Reply:
I like Requirements + RequiredBy!

I struggled with these names.